modules/wallet.go

```diff
@@ -207,7 +207,7 @@ type (
 		// builder. Items are returned by index.
 		ViewAdded() (newParents, siacoinInputs, siafundInputs, transactionSignatures []int)
 
-		// Drop indicates that a transaction is no longer useful, will not be
+		// Drop indicates that a transaction is no longer useful and will not be
```

Comment:
ah, good catch. I was interpreting the last item as speaking about the transaction directly, not its outputs.